canonical-data-syncer-ci/fetch-canonical_data_syncer

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
Whoa this is completely different to the fetch-script I knew :D 

Since this will likely only ever run on ubuntu, you can drop most of it and hardcode the OS, like here: https://github.com/exercism/github-actions/blob/master/configlet-ci/fetch-configlet

You could also drop the token requirement (I've never run into rate limits without token on ubuntu when just doing single calls).

Reply:
The fetch-configlet script was updated a while ago: https://github.com/exercism/configlet/blob/master/scripts/fetch-configlet

The token is not a requirement, it is optional :) But I could indeed drop some of the stuff.

Reply:
I've liberally "borrowed" your code :D

Reply:
I liberally borrowed it from code review feedback in actions/virtual-environments :P